src/infrastructure/commands/status-command.ts

```
@@ -1,7 +1,30 @@
+ * @returns CoCVersion
+ */
+function convertStatusTypeToCoCVersion(type: StatusType): CoCVersion {
+    switch (type) {
+        case 'ver6':
+            return '6';
+        case 'ver7':
+            return '7';
+        default:
+            // TypeScriptの exhaustiveness check
+            const _exhaustiveCheck: never = type;
+            throw new Error(`Unknown status type: ${_exhaustiveCheck}`);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)